awx/api/views.py

```diff
@@ -4529,100 +4566,73 @@ class UnifiedJobStdout(RetrieveAPIView):
                         renderers.JSONRenderer, DownloadTextRenderer, AnsiDownloadRenderer]
     filter_backends = ()
     new_in_148 = True
+    deprecated = True
```

Comment:
What does deprecated mean here? Does it mean "this will be removed" or "our UI will stop using this", because I would think the former.

If it is the former, I don't see how that is going to work. Either we keep the relevant logic here as a conical, universal, source of truth about event reconstruction, or we're asking an unlimited number of client scripts to do shoddy work implementing it themselves. While that would unintentionally become a big selling point for tower-cli, it would make AWX worse. If I was using this in a Jenkins integration, I would be very upset to have it taken away and would consider it to be very unreasonable. How can you build an Ansible runner and not support an option for retrieving standard out?

Additionally, this endpoint is used for the download button which is in the UI, and that's a real front-and-center feature.

Reply:
Yea, I could see an argument for removing this - I don't feel strongly either way about what we call it.  The bottom line is that this endpoint is about to get a _lot_ less traffic for users who primarily consume job results via the UI.

Reply:
https://github.com/ansible/tower-cli/issues/400

This is a API client, in practice. There's not an untamable set of requirements, it's ansi (for the console), requesting the new diff on a poll.

Yeah, I could write the logic for that translation layer of job event API to flat text in the client library.

But if I was going to do that, why don't I just come back to the AWX repo and write the same logic here? (and I am totally happy to do that) Then it's solved for everyone.